kb-importer/src/main/java/org/eclipse/steady/kb/command/Import.java

```diff
@@ -38,25 +38,37 @@
 import org.eclipse.steady.kb.util.Metadata;
 import org.eclipse.steady.shared.util.FileUtil;
 import org.eclipse.steady.shared.util.VulasConfiguration;
+import org.eclipse.steady.backend.BackendConnectionException;
 
 /**
  * import command
  */
 public class Import implements Command {
```

Comment:
Better name: "ImportCommand"

Reply:
If the rest of the code uses a similar convention it is probably because it was "ported" from another project (KayBee)
that has the same structure but is written in Go, where the naming conventions are different ("stuttering", such as command.ImportCommand is frowned upon, and command.Import is recommended).

General rule:
- class names are nouns (e.g. `Command, User, Service`)
- method names are verbs (e.g. `Command.execute()`)
- the names of methods returning a boolean start with is... (or has...) (e.g. `Command.isRunning()`)